src/FTPC.jl

```diff
@@ -14,11 +14,11 @@ type RequestOptions
     url::AbstractString
     hostname::AbstractString
 
-    function RequestOptions(;implicit=false, ssl=false,
-            verify_peer=true, active_mode=false, username="",
-            passwd="", url=nothing, hostname="localhost")
+    function RequestOptions(;implicit::Bool=false, ssl::Bool=false,
+            verify_peer::Bool=true, active_mode::Bool=false, username::AbstractString="",
+            passwd::AbstractString="", url::AbstractString="", hostname::AbstractString="localhost")
```

Comment:
`url` and `hostname` are slightly different. I think we allow `hostname` because it is easier for the user and we can create the `url` with that information. Or, a user can pass a `url` to specify the starting directory and maybe more info.